backend/lib/plugins/error-handler.js

```diff
@@ -0,0 +1,43 @@
+const fp = require("fastify-plugin");
+const Sentry = require("@sentry/node");
+
+// Default Fastify error handler
+// https://github.com/fastify/fastify/blob/5bc406d4c6677131237f85b6ae69fe9e34d4b8dd/lib/context.js#L29
+function defaultErrorHandler(error, request, reply) {
```

Comment:
isn't this also conflicting with the `fastify-sensible` plugin which also includes an override for the default error handler?

Reply:
I didn't realize that `fastify-sensible` has an override for the default error handler. i will fix this to use their error handler.